editor/src/components/navigator/navigator-item/navigator-item-dnd-container.tsx

```diff
@@ -134,13 +141,7 @@ function onHover(
     const targetAction = propsOfDraggedItem.highlighted
       ? []
       : [EditorActions.setHighlightedView(propsOfDraggedItem.elementPath)]
-    const canReparent = propsOfDraggedItem
-      .getDragSelections()
-      .every(
-        (dragSelectedItem: DragSelection) =>
-          !EP.pathsEqual(propsOfDraggedItem.elementPath, dragSelectedItem.elementPath) &&
```

Comment:
We probably still want this specific line to prevent accidentally dropping some selection onto one of the selected elements

Reply:
(Except it should definitely be checking against the drop target...)

Reply:
It checking the wrong thing was at the root of this not working correctly.

Reply:
Fixed in 28f7c8d06.